code/script.js

```diff
@@ -1,47 +1,180 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
-
+const inputWrapper = document.getElementById('input-wrapper')
+const nameInput = document.getElementById('name-input')
+const form = document.getElementById ('name')
 // Global variables, if you need any, declared here
 
 // Functions declared here
+// Starts here
```

Comment:
You have a good amount of comments in your code which is making it easy for us to follow